src/io/stakater/pods/Pod.groovy

```diff
@@ -6,17 +6,21 @@ def setToolsImage(Map parameters = [:], String image) {
     parameters.podContainers.defaultContainer.image = image
 }
 
-def mountDockerSocket(Map parameters = [:]) {
-    setPodVolumes(parameters)
-    parameters.podVolumes.isDockerMount = true
-}
-
 Map setDockerConfig(Map parameters = [:]) {
-    mountDockerSocket(parameters)
+    setPodVolumes(parameters)
     setDefaultContainerEnvVarsConfig(parameters)
 
-    parameters.podVolumes.isDockerConfig = true
-    parameters.podContainers.defaultContainer.envVarsConfig.isDocker = true
+    // Don't mount docker socket if useBuildah is true or isDockerMount is true
+    if (!((parameters.containsKey("useBuildah") && parameters.get('useBuildah', true )) ||
```

Comment:
`useBuildah` shouldn't affect the mounting of docker socket. Only `isDockerMount` should be enough. Also the comment is wrong.
If `isDockerMount` is true then docker socket should be mounted and that should be the only condition.

Reply:
`useBuildah` could instead set `parameters.isDockerMount` to `false` if needed.

Reply:
Handled